src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -210,6 +210,8 @@ export const Spotlight: FC = () => {
       <Response>
         {contacting ? (
           loader
+        ) : response === null ? (
```

Comment:
any chance you can break it down to not have two ternary operator next to each other?

Reply:
just hard to read (for me personally)